public/index.html

```diff
@@ -1,55 +1,53 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
     <title>Create Espeo App</title>
-    <base href="/">
+    <base href="/" />
 
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <!-- Chrome for Android theme color -->
-    <meta name="theme-color" content="#fff">
+    <meta name="theme-color" content="#fff" />
     <!-- Add to homescreen for Chrome on Android -->
-    <meta name="mobile-web-app-capable" content="yes">
-    <meta name="application-name" content="Espeo CRA">
-    <link rel="icon" sizes="192x192" href="images/icon-192x192.png">
+    <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="application-name" content="Espeo CRA" />
+    <link rel="icon" sizes="192x192" href="images/icon-192x192.png" />
 
     <!-- Add to homescreen for Safari on iOS -->
-    <meta name="apple-mobile-web-app-capable" content="yes">
-    <meta name="apple-mobile-web-app-status-bar-style" content="black">
-    <meta name="apple-mobile-web-app-title" content="Espeo CRA">
-    <link rel="apple-touch-icon" href="images/icon-192x192.png">
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+    <meta name="apple-mobile-web-app-title" content="Espeo CRA" />
+    <link rel="apple-touch-icon" href="images/icon-192x192.png" />
 
     <!-- Tile for Win8 -->
-    <meta name="msapplication-TileColor" content="#fff">
-    <meta name="msapplication-TileImage" content="images/icon-192x192.png">
+    <meta name="msapplication-TileColor" content="#fff" />
+    <meta name="msapplication-TileImage" content="images/icon-192x192.png" />
 
-    <link rel="icon" type="image/x-icon" href="images/favicon.ico">
+    <link rel="icon" type="image/x-icon" href="images/favicon.ico" />
 
-    <link rel="manifest" href="manifest.json">
-</head>
+    <link rel="manifest" href="manifest.json" />
+  </head>
 
-<body>
+  <body>
     <div id="root"></div>
     <script>
-        (function() {
-            if (!('serviceWorker' in navigator)) {
-                console.log('Service worker not supported');
-                return;
-            }
-            navigator.serviceWorker.register('service-worker.js')
-                .then(function(registration) {
-                    console.log('SW successfully registered');
-                })
-                .catch(function(error) {
-                    console.log('registration failed', error);
-                });
-        })();
+      // (function() {
```

Comment:
@tomfelix please investigate why enabled service worker results in images load crashing, it cannot be released with such bug